CMakeLists.txt

```diff
@@ -179,6 +179,10 @@ if("${SERENITY_ARCH}" STREQUAL "aarch64")
     add_compile_options(-Wno-type-limits)
 endif()
 
+if("${SERENITY_ARCH}" STREQUAL "riscv64")
+    add_compile_options(-march=rv64gc)
```

Comment:
I haven't been paying quite enough attention in the #riscv channel as I probably should, but why can't we set this in the toolchain file?

Reply:
If you mean the [...]Toolchain.txt.in files, we *could*, but we want to enforce using the same ISA (and even ISA spec version, if it becomes relevant) on both toolchains.

Reply:
Sure, and we can do that by having the same contents in both files and a comment saying "oy m8, update the other one". Putting it in the toolchain file also makes sure that any CMake-using ports use the same arch flags as the OS itself.

Reply:
Ah, didn't realize that ports also pick this up.